app/src/components/HomePage/ActionCards.tsx

```diff
@@ -0,0 +1,74 @@
+import type { TFunction } from "i18next";
+import { Box, Card, CardBody, CardHeader, Text } from "@chakra-ui/react";
+import NextLink from "next/link";
+import { MdOutlineAddchart } from "react-icons/md";
+import { Trans } from "react-i18next/TransWithoutContext";
+import DownloadButton from "@/components/HomePage/DownloadButton";
+import type { CityAttributes } from "@/models/City";
+import type { InventoryAttributes } from "@/models/Inventory";
+
+export function ActionCards(props: {
+  inventoryId: string | null;
+  t: TFunction;
+  city?: CityAttributes;
+  lng: string;
+  inventory?: InventoryAttributes;
+}) {
+  return (
+    <Box className="flex gap-[24px] ">
+      <NextLink
+        data-testid={"add-data-to-inventory-card"}
+        href={`/${props.inventoryId}/data`}
+      >
+        <Card
+          shadow="2dp"
+          backgroundColor="base.light"
+          borderColor="interactive.accent"
+          borderWidth="thin"
+          className="h-[132px] hover:shadow-xl"
```

Comment:
Can we remove the hardcoded height and use padding and font size to determine it instead? Makes it easier to adjust to different languages, font zoom factors the user has configured etc.

Reply:
if we use font size, the size of the two cards (one here, one defined inside "Download Button") might end up the wrong size and we want them to be the same size. 
![image](https://github.com/user-attachments/assets/d072bc03-6d62-4693-9320-cbe6ff9182d0)

Also this was just a refactor, I don't want to mess around with the existing working code too much